apps/web/src/components/HyloEditor/HyloEditor.js

```diff
@@ -34,100 +34,124 @@ const HyloEditor = React.forwardRef(({
   const { t } = useTranslation()
   const editorRef = useRef(null)
   const [selectedLink, setSelectedLink] = useState()
-  const editor = useEditor({
-    content: contentHTML,
 
-    extensions: [
-      // Key events respond are last extension first, these will be last
-      Extension.create({
-        name: 'KeyboardShortcuts',
-        // Keep around for debugging for now:
-        // onTransaction: ({ editor, transaction }) => {
-        //   console.log('!!!!! looking how to get all link marks', transaction)
-        //   transactions.doc.node.forEach(child => {
-        //     const [fontSizeMark] = child.marks.filter((m: Mark) => m.type === markType)
-        //    })
-        // },
-        addKeyboardShortcuts () {
-          return {
-            Enter: ({ editor }) => {
-              if (!onEnter) return false
-              return onEnter(editor.getHTML())
-            },
-            Escape: () => {
-              if (!onEscape) return false
-              return onEscape()
-            }
+  const extensions = [
```

Comment:
Moved extensions into their own variable for clarity, but not changes in this area